en/docs/references/connectors.md

```diff
@@ -0,0 +1,5 @@
+# Connectors
```

Comment:
Since we don't use the term connectors anywhere in the Choreo UI, we might have to decide whether it makes sense to use this term here.

Reply:
As discussed and confimed with NuwanD we will keep this term.